sql/moz-fx-data-shared-prod/google_ads_derived/ad_groups_v1/query.sql

```diff
@@ -0,0 +1,23 @@
+WITH ad_group_names AS (
+  SELECT
+    campaign_id,
+    id AS ad_group_id,
+    name AS ad_group_name,
+    ROW_NUMBER() OVER (PARTITION BY id, campaign_id ORDER BY updated_at DESC) AS rn,
+  FROM
+    `moz-fx-data-bq-fivetran`.ads_google_mmc.ad_group_history
+  QUALIFY
+    rn = 1
```

Comment:
```suggestion
    ROW_NUMBER() OVER (PARTITION BY id, campaign_id ORDER BY updated_at DESC) = 1
```

Reply:
I'm actually going to switch this to what I did for `campaigns_v1` and call it `is_most_recent_record`